### DISCUSSION TITLE
Impersonate users

### DISCUSSION THREAD

**Discussion by httpdennis** [nextauthjs/next-auth#3598] [Help]
### Question 💬

Is there a why to impersonate users?
I am using oauth2 as provider.
If yes, how can i do it?

### How to reproduce ☕️

nothing to reproduce.

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
Please search older issues/discussions before opening new ones: https://github.com/nextauthjs/next-auth/discussions/2947
